setup.py

```diff
@@ -33,55 +33,51 @@ def find_files(pattern, path=None, root="rez"):
-        paths += [x[len(basepath):].lstrip(os.path.sep) for x in files]
+        paths += [x[len(basepath) :].lstrip(os.path.sep) for x in files]
 
     return paths
 
 
 this_directory = os.path.abspath(os.path.dirname(__file__))
-with open(os.path.join(this_directory, 'README.md')) as f:
+with open(os.path.join(this_directory, "README.md")) as f:
     long_description = f.read()
 
 
 setup(
     name="rez",
     version=_rez_version,
-    description=("A cross-platform packaging system that can build and "
-                 "install multiple version of packages, and dynamically "
-                 "configure resolved environments at runtime."),
+    description=(
+        "A cross-platform packaging system that can build and "
+        "install multiple version of packages, and dynamically "
+        "configure resolved environments at runtime."
+    ),
     keywords="package resolve version build install software management",
     long_description=long_description,
-    long_description_content_type='text/markdown',
+    long_description_content_type="text/markdown",
     url="https://github.com/AcademySoftwareFoundation/rez",
     author="Allan Johns",
     author_email="nerdvegas@gmail.com",
     maintainer="Contributors to the rez project",
     maintainer_email="rez-discussion@lists.aswf.io",
     license="Apache-2.0",
     license_files=["LICENSE"],
-    entry_points={
-        "console_scripts": get_specifications().values()
-    },
+    entry_points={"console_scripts": get_specifications().values()},
     include_package_data=False,
     zip_safe=False,
-    package_dir={'': 'src'},
-    packages=find_packages('src', exclude=["build_utils",
-                                           "build_utils.*",
-                                           "tests"]),
+    package_dir={"": "src"},
+    packages=find_packages("src", exclude=["build_utils", "build_utils.*", "tests"]),
     package_data={
-        'rez':
-            ['utils/logging.conf'] +
-            ['README*'] +
-            find_files('*', 'completion') +
-            find_files('*', 'data') +
-            find_files('*.exe', 'vendor/distlib'),
-        'rezplugins':
-            find_files('*.cmake', 'build_system', root='rezplugins') +
-            find_files('*', 'build_system/template_files', root='rezplugins'),
-        'rezgui':
-            find_files('rezguiconfig', root='rezgui') +
-            find_files('*', 'icons', root='rezgui')
+        "rez": ["utils/logging.conf"]
+        + ["README*"]
+        + find_files("*", "completion")
+        + find_files("*", "data")
+        + find_files("*.exe", "vendor/distlib"),
+        "rezplugins": find_files("*.cmake", "build_system", root="rezplugins")
+        + find_files("*", "build_system/template_files", root="rezplugins"),
+        "rezgui": find_files("rezguiconfig", root="rezgui")
+        + find_files("*", "icons", root="rezgui"),
     },
+    install_requires=["setuptools"],
```

Comment:
I think we should call-out setuptools here, most likely.
https://github.com/AcademySoftwareFoundation/rez/blob/main/docs/source/installation.rst#installation-script
(also update the supported python versions, and wherever we do the python badges? maybe that is handled automatically when we update the pip package)